build.gradle

```diff
@@ -53,19 +51,13 @@ dependencies {
     compile "org.hibernate:hibernate-core:5.1.16.Final"
     compile "org.grails.plugins:views-json"
     compile "org.grails.plugins:views-json-templates"
-    console "org.grails:grails-console"
+    console("org.grails:grails-console") {
```

Comment:
why do we add this library?

Reply:
not sure, one of the auto-added, but we kept it for applications like transmart or data-showcase. Do you think it could be removed?

Reply:
sorry, I meant `org.fusesource.jansi`